### DISCUSSION TITLE
Vector notation in Typst

### DISCUSSION THREAD

**Discussion by Satheeshsk369** [typst/typst#3199] [General]
### Description

`math.attach(content, t:arrow.r)` when i use this, the compiler always put this in the right top but i want the exact top. This also happen when i use summation notion `sum_(i=1)^N` , this attach on top right and bottom right but not exact top and bottom. What i need to do to solve this issue. 

### Typst version

- [X] I am using the latest version of Typst

**Comment by laurmaedje** [comment]
In inline equations (e.g. `$sum_(i=0)^n$`), attachments are put to the top/bottom right. For block equations (e.g. `$ sum_(i=0)^n $`, note the extra spaces), they are put to the exact top and bottom. You can use the [`limits`](https://typst.app/docs/reference/math/attach#functions-limits) and [`scripts`](https://typst.app/docs/reference/math/attach#functions-scripts) functions to override this behaviour.

**Reply by Satheeshsk369** [reply]
**Thank you**
